### PR TITLE
Added flow SetupClusterAlias for creating a cluster alias in graphite

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#888
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
LGTM

File: tendrl/monitoring_integration/flows/setup_cluster_alias/__init__.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        if not os.path.exists(alias_dir_path):
+            try:
+                os.makedirs(str(alias_dir_path))
+            except OSError as ex:
+                raise FlowExecutionFailedError(
+                    "Failed to create cluster alias dir: (%s)"
+                    " .Error: (%s)" %
+                    (str(alias_dir_path), ex)
+                )
+        alias_path = "%s/%s" % (alias_dir_path, short_name)
```
Comment (nthomas-redhat) [inline tendrl/monitoring_integration/flows/setup_cluster_alias/__init__.py]: Where this var is used?

Comment (shtripat) [reply tendrl/monitoring_integration/flows/setup_cluster_alias/__init__.py]: Oh its not required as below its being created again. will remove this

**Comment by shtripat** [comment]
@cloudbehl plz review

**Pull request merged**